tools/regen_flatbuffers_headers.sh

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env bash
+
+# Builds flatc and runs it on flatbuffer files
+
+TOP_LEVEL=$(dirname $(dirname $(readlink -f ${BASH_SOURCE})))
+# Determine build dir in typical location <top_level>/build/flatbuffers
+BUILD_DIR=${TOP_LEVEL}/build/flatbuffers
+
+mkdir -p $BUILD_DIR
+cd $BUILD_DIR
+cmake $TOP_LEVEL/third_party/flatbuffers
+ninja
+cd $TOP_LEVEL/csrc/serde
+$BUILD_DIR/flatc --cpp ./python_fusion_cache.fbs
```

Comment:
@rdspring1 FYI I added this. It's a little hokey, but might be useful for adding new ops to frontend

Reply:
If you're using the top-level build, https://github.com/NVIDIA/Fuser/pull/238 will also do this.
It could be useful for the stand-alone build.